sdk/frontdoor/Azure.ResourceManager.FrontDoor/api/Azure.ResourceManager.FrontDoor.netstandard2.0.cs

```diff
@@ -608,6 +484,30 @@ public partial class ForwardingConfiguration : Azure.ResourceManager.FrontDoor.M
         public string CustomForwardingPath { get { throw null; } set { } }
         public Azure.ResourceManager.FrontDoor.Models.FrontDoorForwardingProtocol? ForwardingProtocol { get { throw null; } set { } }
     }
+    public partial class FrontDoorBackend
+    {
+        public FrontDoorBackend() { }
+        public string Address { get { throw null; } set { } }
```

Comment:
maybe this is an ip address?

Reply:
According to the swagger, this might be IP address or FQDN, so I'd like to leave it as it is.